.changes/README.md

```diff
@@ -0,0 +1,46 @@
+# Changes
+
+This stores changes to automatically generate the changelog, to avoid merge conflicts. Files should be in a JSON format, with the following format:
+
+```json
+{
+    "description": "single-line description to add to the CHANGELOG.",
+    "issues": [894],
+    "type": "added",
+    "breaking": false
+}
+```
+
+Valid types are:
+- added (Added)
+- changed (Changed)
+- fixed (Fixed)
+- removed (Removed)
+- internal (Internal)
+
+`breaking` is optional and defaults to false. if `breaking` is present for any active changes, we will not allow a minor (or pre-1.0) patch version to be generated. `issues` is also optional, and is currently unused, and is an array of issues fixed by the PR, and defaults to an empty array.
```

Comment:
This `breaking` comment doesn't seem to be implemented.

Reply:
It isn't, neither is `issues` right now. I'll remove it from the documentation probably. The idea was nice, but it's not reported in the generated CHANGELOG and is probably too noisy to be useful.